packages/core-api/src/extensions/ErrorBoundary.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+export type ErrorBoundaryFallbackProps = {
+  error: Error;
+  resetErrorBoundary: () => void;
```

Comment:
To clarify the link to https://www.npmjs.com/package/react-native-error-boundary I'm thinking we use `resetError` here